cli/cli.go

```diff
@@ -164,6 +165,11 @@ func (h *AddressInput) Set(value string) error {
 	return nil
 }
 
+// Type return a string representation of AddressInput.
+func (h *AddressInput) Type() string {
+	return "AddressInput"
```

Comment:
Users won't know what `AddressInput` means. Let's replace with the syntax for it.
Please remove the syntax example from the help message

```suggestion
	return "[<user>@][<host>]:<port>"
```